exporter/jaegerexporter/factory.go

```diff
@@ -38,11 +38,17 @@ func NewFactory() component.ExporterFactory {
 }
 
 func createDefaultConfig() configmodels.Exporter {
+	// TODO: Enable the queued settings.
```

Comment:
What's the blocker for this?

Reply:
Trying to keep PRs focused, and also I need to deprecate the queue retry processor before enabling this on all the exporters